### ISSUE TITLE
Privacy Badger Breaks Korean Address Finding Service

### ISSUE THREAD

**Issue by ASHApple** [EFForg/privacybadger#2826]
#### What is your browser and browser version?
Firefox 96.0b10 (64-bit) Linux x64

#### What is broken and where?
Privacy badger is blocking Daum(Kakao)'s address finding service.
Daum providing address finding service to many website.
This service by commonly using by iframe.

[Daum postcode service guide (Korean)](https://postcode.map.daum.net/guide)

#### What is the "culprit" domain?
`postcode.map.daum.net`

#### What is your debug output for this domain?
```
**** ACTION_MAP for daum.net
daum.net {
  "userAction": "",
  "dnt": false,
  "heuristicAction": "block",
  "nextUpdateTime": 0
}
**** SNITCH_MAP for daum.net
daum.net [
  "lotteon.com",
  "interpark.com",
  "thairath.co.th"
]
```

Let me know if I can privde any other infomation.

**Comment by ghostwords** [comment]
Hello and thank you for getting in touch!

I believe the "culprit" domains here belong to `kakao.com`. We need to teach Privacy Badger that `daum.net` and `kakao.com` belong to the same entity. This should be fixed in the next Privacy Badger update by 0fd09f557d361a940ebf941de2c3615ef7d92f5a.

**Comment by ghostwords** [comment]
The fix for this issue is finally live in Privacy Badger version 2022.9.27. Thanks again for reporting!
